vjunosevolved/docker/launch.py

```diff
@@ -1,38 +1,46 @@
 #!/usr/bin/env python3
 
+import crypt
```

Comment:
Hi @vista- 

since crypt is deprecated now in Py3, would you be willing to check if sha512 hash can be done with smth like

```py
from passlib.hash import sha512_crypt

# create SHA-512 hash of the password
password_hash = sha512_crypt.hash("admin@123")
```

This will require `pip install passlib`

Reply:
Done!